### PR TITLE
VZ-9204: Docs for SMTP configuration in Verrazzano

### PR DESCRIPTION
**Pull request by lake-of-dreams**
Docs for changes done in https://github.com/verrazzano/verrazzano/pull/5844 to support SMTP configuration in Grafana

### CODE REVIEW

File: content/en/docs/customize/grafana.md [hunk 1]
```diff
@@ -59,4 +61,56 @@ If you prefer to use your own Grafana database, complete the following steps:
 </div>
 {{< /clipboard >}}
 
+### Configure SMTP server
+
+To configure SMTP server for Grafana notifications, complete the following steps:
```
Comment (kminder) [inline content/en/docs/customize/grafana.md:66]: This should be more like the below right?
To configure Grafana to send SMTP notifications, complete the following steps:

File: content/en/docs/customize/grafana.md [hunk 2]
```diff
@@ -59,4 +61,56 @@ If you prefer to use your own Grafana database, complete the following steps:
...
+           skipVerify: true
+           startTLSPolicy: ""
+           userKey: "username"
+           keyFileKey: "key"
+   ```
+
+</div>
+{{< /clipboard >}}
+
+For more information abou the Grafana SMTP configurations, see [Grafana Documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#smtp).
```
Comment (kminder) [inline content/en/docs/customize/grafana.md:114]: about

File: content/en/docs/reference/API/vco-clusters-v1alpha1.md [hunk 3]
```diff
@@ -503,6 +503,17 @@ string
 </tr>
 <tr>
 <td>
+<code>thanosQueryStore</code></br>
+<em>
+string
+</em>
+</td>
+<td>
+<p>The Thanos Query Store API host name for this managed cluster.</p>
```
Comment (kminder) [inline content/en/docs/reference/API/vco-clusters-v1alpha1.md:512]: How did the end up in this PR?

File: content/en/docs/reference/API/vpo-verrazzano-v1alpha1.md [hunk 4]
```diff
@@ -491,6 +491,37 @@ string
 </tr>
 </tbody>
 </table>
+<h3 id="install.verrazzano.io/v1alpha1.CAPIComponent">CAPIComponent
```
Comment (kminder) [inline content/en/docs/reference/API/vpo-verrazzano-v1alpha1.md:494]: Why is this in this PR?

File: content/en/docs/reference/API/vco-clusters-v1alpha1.md (hunk 3 above)
Comment (lake-of-dreams) [reply content/en/docs/reference/API/vco-clusters-v1alpha1.md:512]: it seems some api changes from past were not propagated, https://github.com/verrazzano/docs/tree/master/content/en/docs/reference/API all files seems at least couple months old

File: content/en/docs/reference/API/vpo-verrazzano-v1alpha1.md (hunk 4 above)
Comment (lake-of-dreams) [reply content/en/docs/reference/API/vpo-verrazzano-v1alpha1.md:494]: same as [above](https://github.com/verrazzano/docs/pull/1036/files#r1182079220)

File: content/en/docs/customize/grafana.md (hunk 1 above)
Comment (lake-of-dreams) [reply content/en/docs/customize/grafana.md]: fixed

File: content/en/docs/customize/grafana.md (hunk 2 above)
Comment (lake-of-dreams) [reply content/en/docs/customize/grafana.md]: fixed

**Comment by rosemarymarano** [comment]
Waiting for approval from @kminder before the literary review/edit.

**Review by kminder — APPROVED** [review]
Approved.

File: content/en/docs/customize/grafana.md [hunk 5]
```diff
@@ -6,6 +6,8 @@ Weight: 5
 draft: false
 ---
 
+### Configure Grafana database
+
 By default, Verrazzano automatically installs and configures a Grafana database. However, you can use your own external database.
 
 If you prefer to use your own Grafana database, complete the following steps:
```
Comment (rosemarymarano) [inline content/en/docs/customize/grafana.md:13]: Copying the example in Step 1, includes the comments. Please fix.

File: content/en/docs/customize/grafana.md [hunk 6]
```diff
@@ -6,6 +6,8 @@ Weight: 5
 draft: false
 ---
 
+### Configure Grafana database
```
Comment (rosemarymarano) [inline content/en/docs/customize/grafana.md:9]: -> Configure a Grafana database

File: content/en/docs/customize/grafana.md [hunk 7]
```diff
@@ -59,4 +61,56 @@ If you prefer to use your own Grafana database, complete the following steps:
 </div>
 {{< /clipboard >}}
 
+### Configure SMTP server
```
Comment (rosemarymarano) [inline content/en/docs/customize/grafana.md:64]: -> Configure an SMTP server

File: content/en/docs/customize/grafana.md [hunk 8]
```diff
@@ -59,4 +61,56 @@ If you prefer to use your own Grafana database, complete the following steps:
...
+           skipVerify: true
+           startTLSPolicy: ""
+           userKey: "username"
+           keyFileKey: "key"
+   ```
+
+</div>
+{{< /clipboard >}}
+
+For more information about the Grafana SMTP configurations, see [Grafana Documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#smtp).
```
Comment (rosemarymarano) [inline content/en/docs/customize/grafana.md:114]: -> For more information about Grafana SMTP configurations, see the [Grafana Documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#smtp).

**Review by rosemarymarano — COMMENTED** [review]
Few edits to fix.

File: content/en/docs/customize/grafana.md (hunk 5 above)
Comment (lake-of-dreams) [reply content/en/docs/customize/grafana.md:13]: removed comments

File: content/en/docs/customize/grafana.md (hunk 6 above)
Comment (lake-of-dreams) [reply content/en/docs/customize/grafana.md]: fixed

File: content/en/docs/customize/grafana.md (hunk 7 above)
Comment (lake-of-dreams) [reply content/en/docs/customize/grafana.md]: fixed

File: content/en/docs/customize/grafana.md (hunk 8 above)
Comment (lake-of-dreams) [reply content/en/docs/customize/grafana.md]: fixed

**Pull request merged**